oak_functions/examples/mobilenet/config.toml

```diff
@@ -0,0 +1,11 @@
+lookup_data_url = ""
+lookup_data_download_period = "10m"
+worker_threads = 4
+
+[tf_model]
+url = "https://storage.googleapis.com/oak-tensorflow-models/mobilenet_v2_1.4_224_frozen"
```

Comment:
We should not rely on external resources, could you copy this in third_party and refer to it as a file? Also even in production, we would not want the loader to go and fetch additional files (apart from the key / value entries, by design), they should all be bundled in the Docker image.